tjmonopix2/system/fifo_readout.py

```diff
@@ -38,6 +38,13 @@ class StopTimeout(Exception):
 
 
 class FifoReadout(object):
+    """Basic class for FIFO functionality.
+
+    Parameters
+    ----------
+    object : class
+        Main DAQ object
+    """
```

Comment:
`object` can be removed as this is old-style python syntax and not required anymore. Therefore, the parameter can be removed. "Basic" is also misleading, I think, as this is neither a base class nor a naive implementation. It is a class dealing with all things FIFO-related.